tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -2041,6 +2041,35 @@ def func(self, xp, a):
         return xp.arctan(a)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': (
+            _make_same_in_out_dtypes(2, chainerx.testing.float_dtypes)),
+        'input_lhs': ['random', 3.14, 1.57, 100., -100.],
+        'input_rhs': ['random', 3.14, 1.57, 100., -100.],
```

Comment:
`3.14` and `1.57` seem not appropriate for the inputs of `arctan2` because `arctan2` does not take angle values.

Reply:
Oh right.